config/rhtapwatch.yaml

```diff
@@ -7,7 +7,10 @@ team_automation:
     Epic:
       # collector: get_issues
       rules:
-        - check_parent_link
+        - rule: check_parent_link
+          kwargs:
+            # This is a CEL expression
+            ignore: '"orphan" in .labels'
```

Comment:
Is this label only utilized in the RHTAPWATCH project or should we apply this rule across other projects too?

Reply:
It's only for you. You can pick the label you want and just set it here in the CEL expression. Your choice!